configs/projects/puppet-agent.rb

```diff
@@ -92,6 +92,7 @@
   proj.component "puppet"
   proj.component "facter"
   proj.component "hiera"
+  proj.component "leatherman"
   proj.component "marionette-collective"
   proj.component "cpp-pcp-client"
   proj.component "pxp-agent"
```

Comment:
Derp, that's what you say in your commit message. I should have read that first...